jinahub/indexers/query/vector/AnnoyIndexer/__init__.py

```diff
@@ -17,43 +18,66 @@ class AnnoyIndexer(BaseNumpyIndexer):
         Annoy package dependency is only required at the query time.
     """
 
-    def __init__(self, metric: str = 'euclidean', n_trees: int = 10, search_k: int = -1, *args, **kwargs):
+    def __init__(
+            self,
+            top_k: int = 10,
+            num_dim: int = 768,
```

Comment:
if u do not have any power over how many `dimensions` the `indexes` have I do not see why it should be a parameter.

This should be extracted from the `dump` vectors

Reply:
ah yes, this should be changed